Makefile

```diff
@@ -144,7 +144,7 @@ run: manifests generate fmt vet ## Run a controller from your host.
 
 .PHONY: docker-build
 docker-build: test ## Build docker image with the manager.
-	podman build -t ${IMG} .
+	podman build --build-arg GOWORK=$(GOWORK) ${DOCKER_BUILD_ARGS} -t ${IMG} .
```

Comment:
Might need to declare the `DOCKER_BUILD_ARGS` above: https://github.com/openstack-k8s-operators/keystone-operator/pull/364/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52